web/ui/react-app/src/components/CustomInfiniteScroll.tsx

```diff
@@ -0,0 +1,50 @@
+import { ComponentType, useEffect, useState } from 'react';
+import InfiniteScroll from 'react-infinite-scroll-component';
+
+const initialNumberOfTargetsDisplayed = 50;
```

Comment:
If this is meant to be a general component and not just for targets, then this should be about "items", not "targets".